azure-pipelines.yml

```diff
@@ -58,13 +58,13 @@ steps:
       projects: '**/*.UnitTests.csproj'
 
   - task: DotNetCoreCLI@2
-    displayName: "Publish - dotnet publish $(__SolutionName__)"
+    displayName: "Publish Apprentice Portal Web"
     inputs:
       command: publish
       publishWebProjects: false #set to false as this setting (which defaults to true) will parse the entire repo for web projects
-      projects: "src/$(__SolutionName__)/$(__SolutionName__).csproj"
+      projects: src/**/*.csproj
       arguments: "-o $(build.artifactstagingdirectory)/publish -c $(BuildConfiguration) --no-build"
-      zipAfterPublish: false
+      zipAfterPublish: true
```

Comment:
You could remove this line as `zipAfterPublish` will be true by default if not present.